gmm_client/src/client_test_helper.erl

```diff
@@ -68,3 +98,22 @@ check_permission_result(Result)->
 check_permissions(From, To)->
   {ok, _, Body, _} = client:get_edge_permissions(From, To),
   check_permission_result(Body).
+
+get_list_from_string(ListString) ->
```

Comment:
Ta funkcje jest strasznie nieczytelna. Trzeba to okomentować, zmienić nazwę, i prawdopodobnie całkiem przerobić